frontend/public/components/masthead.tsx

```diff
@@ -1,30 +1,146 @@
 import * as React from 'react';
+import * as classNames from 'classnames';
 
 import * as _ from 'lodash-es';
 import { Link } from 'react-router-dom';
 import * as okdLogoImg from '../imgs/okd-logo.svg';
 import * as ocpLogoImg from '../imgs/openshift-platform-logo.svg';
 import * as onlineLogoImg from '../imgs/openshift-online-logo.svg';
 import * as dedicatedLogoImg from '../imgs/openshift-dedicated-logo.svg';
+import * as rhLogoImg from '../imgs/redhat-logo-modal.svg';
+import * as okdModalImg from '../imgs/okd-logo-modal.svg';
 import { FLAGS, connectToFlags, flagPending } from '../features';
 import { authSvc } from '../module/auth';
-import { Dropdown, ActionsMenu } from './utils';
+import { Dropdown, ActionsMenu, AsyncComponent } from './utils';
+import { openshiftHelpBase } from './utils/documentation';
+import { k8sVersion } from '../module/status';
+import { createModalLauncher } from './factory/modal';
 
 import { coFetchJSON } from '../co-fetch';
 import { SafetyFirst } from './safety-first';
 
 const developerConsoleURL = (window as any).SERVER_FLAGS.developerConsoleURL;
 
+const PFAboutModal = (props) => <AsyncComponent loader={() => import('patternfly-react').then(c => c.AboutModal)} {...props} />;
+const PFAMVersions = (props) => <AsyncComponent loader={() => import('patternfly-react').then(c => c.AboutModal.Versions)} {...props} />;
+const PFAMVersionItem = (props) => <AsyncComponent loader={() => import('patternfly-react').then(c => c.AboutModal.VersionItem)} {...props} />;
```

Comment:
Does this fire off 3 requests for patternfly-react?

The pattern you want is to put `AboutModal` in a separate module and then load `AboutModal` async. That way you can reference the patternfly-react components directly without async loading from the `AboutModal` component.